src/apis/vaults.ts

```diff
@@ -181,7 +181,7 @@ export class DefaultVaultsAPI {
                     onlyIssued
                 )
                 : await customAPIRPC.vaultRegistry.getCollateralizationFromVault(vaultId, onlyIssued);
-            return this.scaleUsingParachainGranularity(collateralization);
+            return this.scaleUsingParachainGranularity(new Big(collateralization));
```

Comment:
is Big compatible with the u128 parameter of `scaleUsingParachainGranularity`?

Reply:
We could remove the `private` access modifier and write a unit test

Reply:
`scaleUsingParachainGranularity` takes `Big` now

Reply:
yep, let's remember to do that next week :)